lib/python/vdm/server/voltdbserver.py

```diff
@@ -159,7 +159,8 @@ def start_database(self, recover=False):
             response = requests.get(url)
             return create_response(response.text, 200)
         else:
-            return create_response('Start request sent successfully to servers: ' + str(server_status), 200)
+            return create_response('Start request sent successfully to servers: ' +
+                                   str(HTTPListener.json_loads_without_unicode(server_status)), 200)
```

Comment:
I looked into this a bit. It appears like we can use json.dumps(server_status) instead of going through it ourselves and removing the 'u's.